ask_answer_from_chatGPT.py

```diff
@@ -1,13 +1,41 @@
+import openai
+import asyncio
 
-def sending_question(question: str)->str:
-    """
-    The model will send a question to chatGPT
-    """
-    pass
+api_key_file = 'api_key_file.txt'  # Replace with the path to your API key file
+
+with open(api_key_file, 'r') as file:
+    openai.api_key = file.read().strip()
+
+
+async def generate_chat_response( messages):
+    completion =openai.ChatCompletion.create(
+        model="gpt-3.5-turbo",
+        messages=messages
+    )
+    chat_response = completion.choices[0].message.content
+    return chat_response
 
-def getting_answer()->str:
+async def list_question_to_the_chatGPT(questions: list) -> list:
```

Comment:
The right formatt in List (not list) 
which represents the existing List data structure in Python